lib/process-data-availability.js

```diff
@@ -210,7 +210,7 @@ const processDataAvailability = async availabilityReport => {
         if (!acc.expandedResourcesCounts[resourceName]) {
           acc.expandedResourcesCounts[resourceName] = {};
         }
-        acc.expandedResourcesCounts[resourceName][expansion.resourceName] = expansion.numRecordsFetched;
+        acc.expandedResourcesCounts[resourceName][expansion.resourceName] = expansion.numExpandedRecordsFetched;
```

Comment:
This is probably supposed to be `expansion.numUniqueRecordsFetched`?

The data structure is the following:

```
 {
      "resourceName": "Member",
      "dateField": "ModificationTimestamp",
      "dateHigh": "2024-04-11T22:00:28.000Z",
      "dateLow": "2023-04-06T08:40:02.000Z",
      "numUniqueRecordsFetched": 20322,
      "recordCount": 20318,
      "averageResponseBytes": 1237658.746031746,
      "medianResponseBytes": 1273922,
      "stdDevResponseBytes": 186217.73024078758,
      "averageResponseTimeMs": 869.5396825396825,
      "averageResponseTimeMillis": 869.5396825396825,
      "medianResponseTimeMs": 695,
      "stdDevResponseTimeMs": 458.44311500461794,
      "numRecordsFetched": 60946,
      "numSamples": 63,
      "pageSize": 1000,
      "expansions": [
        {
          "resourceName": "Office",
          "dateField": "ModificationTimestamp",
          "dateHigh": "2024-04-11T20:38:54.000Z",
          "dateLow": "2023-04-06T08:39:47.000Z",
          "numUniqueRecordsFetched": 1535,
          "averageResponseBytes": 2394914.1904761903,
          "medianResponseBytes": 2469869,
          "stdDevResponseBytes": 360561.49692190776,
          "averageResponseTimeMs": 1586.1904761904761,
          "averageResponseTimeMillis": 869.5396825396825,
          "medianResponseTimeMs": 1261,
          "stdDevResponseTimeMs": 817.917600870686,
          "numRecordsFetched": 60946,
          "numExpandedRecordsFetched": 60946,
          "numSamples": 63
        }
      ]
    },
```

`numExpandedRecordsFetched` is the total count, but since we sample with different strategies, they're de-duplicated.